File: app/src/main/java/com/example/pizzamaniaapp/app/ui/main/MainActivity.java
package com.example.pizzamaniaapp.app.ui.main;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;
import com.google.android.material.bottomnavigation.BottomNavigationView;
import com.example.pizzamaniaapp.R;
import com.example.pizzamaniaapp.app.ui.cart.CartFragment;
import com.example.pizzamaniaapp.app.ui.home.HomeFragment;
import com.example.pizzamaniaapp.app.ui.menu.MenuFragment;
import com.example.pizzamaniaapp.app.ui.orders.OrdersFragment;

public class MainActivity extends AppCompatActivity {
    @Override protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        BottomNavigationView nav = findViewById(R.id.bottomNav);
        nav.setOnItemSelectedListener(item -> {
            Fragment f;
            int id = item.getItemId();
            if (id == R.id.menu) f = new MenuFragment();
            else if (id == R.id.cart) f = new CartFragment();
            else if (id == R.id.orders) f = new OrdersFragment();
            else f = new HomeFragment();
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.container, f).commit();
            return true;
        });
        nav.setSelectedItemId(R.id.home);
    }
}

File: app/src/main/java/com/example/pizzamaniaapp/app/ui/auth/MainActivity.kt
package com.example.pizzamaniaapp.app.ui.auth

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import com.example.pizzamaniaapp.ui.LoginActivity
import com.example.pizzamaniaapp.R

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)

        // Insets adjuster to avoid overlap with system bars (status/nav bars)
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }

        // Find the LOGIN button (id: button5) from activity_main.xml
        val loginBtn = findViewById<Button>(R.id.button5)

        loginBtn.setOnClickListener {
            // move MainActivity ➝ LoginActivity
            val intent = Intent(this, LoginActivity::class.java)
            startActivity(intent)
        }
    }
}